.github/workflows/maven-test.yml

```diff
@@ -18,5 +18,15 @@ jobs:
         java-version: '11'
         distribution: 'temurin'
         cache: maven
-    - name: Run the Maven verify phase
-      run: mvn -B --update-snapshots verify
+    - name: Run Maven Build Step
+      run: mvn -B --update-snapshots -DskipTests package
+    - name: Run Maven Test Step
+      run: mvn -B --update-snapshots -Dmaven.test.failure.ignore=true test
```

Comment:
Most probably I do not get GitHub actions yet. Isn't it too complicated do run `package` and then `test`? I guest that it runs `compile` twice.

Reply:
We need the package phase for this to work, we do not want to have it in one command with `test` since this would mix the results in one steps and be hard to combine with the required flags.